### DISCUSSION TITLE
version 24.4

### DISCUSSION THREAD

**Discussion by hvianna** [hvianna/audioMotion.js#60] [Announcements]
## What's new

Maintenance update with some improvements and bug fixes.

**If you hated the media panel auto-hide feature before, please try it again!** 😅

### Added:

+ Support for #EXTALB directive in m3u playlists (discussed in [#37](https://github.com/hvianna/audioMotion.js/issues/37#issuecomment-1732597308));
+ Version control to let users know when the app has been updated (the update message should start appearing with the next version).

### Changed & improved:

+ **Media panel auto-hide improvements:** fixed the scrollbar on small windows ([#54](https://github.com/hvianna/audioMotion.js/issues/54)); added a small delay before triggering; avoid changing state when mouse leaves the window;
+ Song metadata is now saved to the stored playqueue and playlists;
+ **Added instructions on how to install the app in the [Getting started](README.md#getting-started) section;**
+ Added links to the docs website and changelog (click the version number) on the Help panel.

### Fixed:

+ File explorer stalling on "Loading..." when folder was not found ([#55](https://github.com/hvianna/audioMotion.js/issues/55));
+ Built-in background images not loading.

<hr /><em>This discussion was created from the release <a href='https://github.com/hvianna/audioMotion.js/releases/tag/v24.4'>version 24.4</a>.</em>

**Comment by JesusGod-Pope666-Info** [comment]
http://audiomotion.app/
